react/src/components/user/course/CourseHero.jsx

```
@@ -0,0 +1,184 @@
+              </div>
+
+            </div>
+          </div>
+          
+          <div className="col-lg-4">
+            <div className="course-card">
+              <div className="course-pricing">
+                <div className="price">
+                  {course.discount_price ? (
+                    <>
+                      <span className="original-price">${parseFloat(course.price).toFixed(2)}</span>
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)